toqito/matrix_ops/outer_product.py

```diff
@@ -3,15 +3,14 @@
 
 
 def outer_product(v1: np.ndarray, v2: np.ndarray) -> np.ndarray:
-    r"""
-    Compute the outer product :math:`|v_1\rangle\langle v_2|` of two vectors.
+    r"""Compute the outer product :math:`|v_1\rangle\langle v_2|` of two vectors.
 
     The outer product is calculated as follows :cite:`WikiOuterProd` :
 
     .. math::
         \left|\begin{pmatrix}a_1\\\vdots\\a_n\end{pmatrix}\right\rangle\left\langle\begin{pmatrix}b_1\\\vdots\\b_n\end{pmatrix}\right|=\begin{pmatrix}a_1\\\vdots\\a_n\end{pmatrix}\begin{pmatrix}b_1&\cdots&b_n\end{pmatrix}=\begin{pmatrix}a_1b_1&\cdots&a_1b_n\\\vdots&\ddots&\vdots\\a_1b_n&\cdots&a_nb_n\end{pmatrix}
 
-    Example
+    Example:
```

Comment:
```suggestion
    Example
```
Ruff quick `---fix` adds a `:` after every section heading. Check if this causes any issues with Sphinx build.